test/Actions.test.ts

```diff
@@ -130,4 +130,210 @@ contract('Actions', ([owner, random]) => {
       assert.equal(depositArgs.index, new BN(index))
     })
   })
+
+  describe('Parse Mint Arguments', () => {
+    it('should not be able to parse a non Mint action', async () => {
+      const actionType = ActionType.OpenVault
+      const asset = ZERO_ADDR
+      const vaultId = '0'
+      const amount = '10'
+      const index = '0'
+      const bytesArgs = ZERO_ADDR
+
+      const data = {
+        actionType: actionType,
+        owner: owner,
+        sender: owner,
+        asset: asset,
+        vaultId: vaultId,
+        amount: amount,
+        index: index,
+        data: bytesArgs,
+      }
+
+      await expectRevert(actionTester.testParseMintAction(data), 'Actions: can only parse arguments for mint actions')
```

Comment:
```suggestion
    it('should not be able to parse a non Mint action', async () => {
      const data = {
        actionType: ActionType.OpenVault,
        owner: ZERO_ADDR,
        sender: owner,
        asset: ZERO_ADDR,
        vaultId: vaultId,
        amount: '10',
        index: '0',
        data: ZERO_ADDR,
      }

      await expectRevert(actionTester.testParseMintAction(data), 'Actions: can only parse arguments for mint actions')
```

Reply:
If we're not using those `const` variables for verification, then we can just declare the testing data in the `data` object, which save us some lines.